embassy-stm32/src/hrtim/traits.rs

```diff
@@ -0,0 +1,200 @@
+use crate::rcc::sealed::RccPeripheral;
+use crate::time::Hertz;
+
+#[derive(Clone, Copy)]
+pub(crate) enum HighResolutionControlPrescaler {
```

Comment:
it's already known it's "HighResolution" now that it's in the `hrtim` module. It could be just "ControlPrescaler"